outputs.tf

```diff
@@ -0,0 +1,7 @@
+output "webapp_url" {
+  value = "https://${azurerm_app_service.main.default_site_hostname}"
+}
+
+output "webhook" {
+  value = "https://${azurerm_app_service.main.site_credential[0].username}:${azurerm_app_service.main.site_credential[0].password}@${azurerm_app_service.main.name}.scm.azurewebsites.net/docker/hook"
```

Comment:
Not vital here, but for a live project I'd be inclined to treat this as sensitive; letting anyone be able to trigger a deployment is unlikely to be a massive security breach, but could let somebody cause problems!